parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageReadStore.java

```diff
@@ -74,7 +74,7 @@
     ColumnChunkPageReader(BytesInputDecompressor decompressor, List<DataPage> compressedPages,
         DictionaryPage compressedDictionaryPage, OffsetIndex offsetIndex, long rowCount) {
       this.decompressor = decompressor;
-      this.compressedPages = new LinkedList<DataPage>(compressedPages);
+      this.compressedPages = new ArrayList<DataPage>(compressedPages);
```

Comment:
Why are we creating a new List of an existing list?

Reply:
I changed this to a Queue because that is the most natural way to remove an item from the head of a List.  Thank you for your interest and indulging me with your time!